terraform/files/bin/create_cluster.sh

```diff
@@ -191,14 +191,25 @@ if test "$DEPLOY_METRICS" = "true"; then
   apply_metrics.sh "$CLUSTER_NAME" || exit $?
 fi
 
+# Harbor settings
+if test ! -s ~/$CLUSTER_NAME/harbor-settings; then
+	cp -p ~/cluster-defaults/harbor-settings ~/$CLUSTER_NAME/
```

Comment:
It seems that there is an indentation made by `tab` and the rest of your code uses 2 spaces instead. Unify this (personally I prefer 2 spaces)

Reply:
Addressed in 3b86ce283d77c80d1b6ee3415379ea1035ef7699